nano/node/websocket.cpp

```diff
@@ -613,6 +752,36 @@ void nano::websocket::listener::on_accept (boost::system::error_code ec)
 	}
 }
 
+std::vector<std::shared_ptr<nano::websocket::session>> nano::websocket::listener::find_sessions (nano::websocket::topic const & topic_a) const
+{
+	std::vector<std::shared_ptr<nano::websocket::session>> sessions_l;
+	nano::lock_guard<std::mutex> lk (sessions_mutex);
+	for (auto & weak_session : sessions)
+	{
+		auto session_ptr (weak_session.lock ());
+		if (session_ptr)
+		{
+			auto subscription_l (session_ptr->subscriptions.find (topic_a));
+			if (subscription_l != session_ptr->subscriptions.end ())
```

Comment:
Could use `if (session_ptr->subscriptions.count (topic_a) > 0)` to save a line as you don't need the iterator and still seems readable.